lib/install/management/store_files.go

```diff
@@ -189,3 +191,44 @@ func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfi
 	}
 	return nil
 }
+
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) {
+	if d.force {
```

Comment:
Ah, good to know. And re: the rest of your comments, I'm refactoring that whole section to get rid of hackish `Split`s etc.